cabal-install/src/Distribution/Client/InstallSymlink.hs

```diff
@@ -150,11 +153,13 @@ symlinkBinaries
                   privateBinDir <- pkgBinDir pkg ipid
                   ok <-
                     symlinkBinary
-                      overwritePolicy
-                      publicBinDir
-                      privateBinDir
-                      (prettyShow publicExeName)
-                      privateExeName
+                      ( Symlink
+                          overwritePolicy
```

Comment:
Stuff like this may be shortened by RecordWildcards but some may say that it's less readable, so up to you.

Reply:
I normally shy away from `-XRecordWildCards` but in this case it looks fine so I went with it. Thanks for suggesting it @ulysses4ever.

Reply:
Right, I'm also cautious about RWC but this looked like a good spot for it. Thank you for acting on the feedback!